### ISSUE TITLE
SwipeableTabBarController inside UINavigationController

### ISSUE THREAD

**Issue by thealzgit** [marcosgriselli/SwipeableTabBarController#35]
I just want to move between my tab, but i dont want to loose my title and nav bar controller. Is it is possible?

**Comment by marcosgriselli** [comment]
I think so, if you put the `UITabBarController` inside a `UINavigationController` you should be able to do that.

**Comment by marcosgriselli** [comment]
Definitely possible 

![tabcontroller_inside_navcontroller](https://user-images.githubusercontent.com/14804033/38375719-3fe5becc-38cd-11e8-98ea-35dafd2297a6.gif)

Quick code sample in `AppDelegate.swift`

```swift 
  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
        // Override point for customization after application launch.
        window = UIWindow(frame: UIScreen.main.bounds)
        let tabBarController = TabBarController()
        
        let controller1 = UIViewController()
        controller1.view.backgroundColor = UIColor.blue
        controller1.title = "VC 1"
        let controller2 = UIViewController()
        controller2.view.backgroundColor = UIColor.red
        controller2.title = "VC 2"
        
        tabBarController.setViewControllers([controller1, controller2], animated: false)
        tabBarController.title = "TabBar Title"
        let navigation = UINavigationController(rootViewController: tabBarController)
        if #available(iOS 11.0, *) { navigation.navigationBar.prefersLargeTitles = true } 
        window?.rootViewController = navigation
        tabBarController.selectedViewController = controller1
        window?.makeKeyAndVisible()
        return true
    }
```
